src/main/java/rcms/utilities/daqexpert/reasoning/logic/basic/TTSDeadtime.java

```diff
@@ -23,6 +26,8 @@
 	public TTSDeadtime() {
         this.name = "TTS Deadtime";
 		this.priority = ConditionPriority.IMPORTANT;
+		this.action = new SimpleAction("Look at CPM page");
```

Comment:
Is this message shown to the shifter if the DAQExpert can not further identify the origin of the TTS backpressure (i.e. not associate it to a particular FED) ? 

We should give more details in the action, e.g. `Look at the CPM page, on the 'Rates and deadtimes' tab, in the 'Latest deadtimes' box to identify the source of the deadtime`.

Reply:
It's the one defined in #141 as `TTS dead time > 2% =>look at CPM page`

Reply:
@mommsen, @hsakulin do you have any opinion whether we should include more detail in the action (maybe the specification in #141 was a bit too short) ? 

If there are no strong opinions we can leave it as is for the moment and expand it later if needed.

Reply:
Look at the CPMController page, on the 'Rates and deadtimes' tab, in the 'Latest deadtimes' box to identify the source of the deadtime.